src/tracker.ts

```diff
@@ -102,10 +102,11 @@ export class Tracker {
       }),
     };
 
-    return fetch(
-      `${this.core.getSettings().endpoint}/com.snowplowanalytics.snowplow/tp2`,
-      props,
-    )
+    return this.core.getManager()
+      .fetch(
```

Comment:
@jonnyparis, webcm doesn't support `manager.fetch()` so i couldn't verify it works good on localhost. could you check this moment?

Reply:
Thanks for flagging, it's a new feature but we'll update webcm soon too. Meanwhile I'll verify this behaviour locally

Reply:
Confirming that this works fine in CF Zaraz ✅
![image](https://user-images.githubusercontent.com/6400000/227274041-74ff86ab-e64e-4b8b-81d6-e5bbcc1c6869.png)